src/Behat/Mink/Driver/CoreDriver.php

```diff
@@ -90,6 +96,28 @@ public function getContent()
      */
     public function find($xpath)
     {
+        $elements = array();
+
+        foreach ($this->findElements($xpath) as $xpath) {
+            $elements[] = new NodeElement($xpath, $this->session);
+        }
+
+        return $elements;
+    }
+
+    /**
+     * Finds elements with specified XPath query.
+     *
+     * @see find()
+     *
+     * @param string $xpath
+     *
+     * @return string[] The XPath of the matched elements
+     *
+     * @throws UnsupportedDriverActionException When operation not supported by the driver
+     */
+    protected function findElements($xpath)
```

Comment:
Well, my issue with the ``findElements`` name is the ``elements`` part, because it does not return elements objects. So ``matchElements`` is not better in this regard